### DISCUSSION TITLE
Schedule cycle tasks accurate to seconds

### DISCUSSION THREAD

**Discussion by bangbangDong** [apache/airflow#43359] [General]
### Description

The crontab expression for the current airflow scheduling task is 5 bits, instead of the usual 6 bits that are accurate to the second

### Use case/motivation

airflow cycle scheduling task crontab does not support seconds, is there any method to support the scheduling task to run the scheduling task at 8:12 minutes 27 seconds every day

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Please use discussions if you want to ask quesitons not issues. When you create an issue you are instructed to use discussions for questions.

You can use custom timetable if you want. Look it up in the docs.

But be aware airlfow is usually used to run fine-grained tasks - that run for multiple seconds. And usually overhead on starting a task is counted in few seconds at least (depending on executor) - so there is no way to get second precision in execution. You are likely trying to use Airflow for something it was not created for.
